src/components/common/Layout/components/Account/components/Notifications/MessagingSection.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  Divider,
+  HStack,
+  Icon,
+  IconButton,
+  Link,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+  useDisclosure,
+} from "@chakra-ui/react"
+import { XMTPProvider } from "@xmtp/react-sdk"
+import Button from "components/common/Button"
+import { Modal } from "components/common/Modal"
+import dynamic from "next/dynamic"
+import { ArrowSquareOut, Gear } from "phosphor-react"
+import SubscriptionPromptSkeleton from "../MessageSkeleton/SubscriptionPromptSkeleton"
+import { SubscriptionPrompt } from "./SubscriptionPrompt"
+import { MessagingWrapper, useMessagingContext } from "./components/MessagingContext"
+import NotificationsSection from "./components/NotificationsSection"
+
+const Messages = dynamic(() => import("./Messages"))
+
+const MessagingSection = () => {
+  const { isOpen, onOpen, onClose } = useDisclosure()
+  const {
+    isRegisteringWeb3Inbox,
+    isSigningWeb3Inbox,
+    isSubscribingWeb3Inbox,
+    isWeb3InboxLoading,
+    web3InboxError,
+    subscribeWeb3Inbox,
+    web3InboxSubscription,
+    hasXmtpAccess,
+    isCheckingXmtpAccess,
+    subscribeXmtp,
+    isLoadingDependencies,
+    isSubscribingXmtp,
+  } = useMessagingContext()
+
+  return (
+    <>
+      <NotificationsSection
+        title={
+          <>
+            Messages
+            <IconButton
+              aria-label="messagingSettings"
+              icon={<Icon as={Gear} boxSize="1.1em" />}
+              variant={"ghost"}
+              borderRadius={"full"}
+              ml={1}
+              onClick={onOpen}
+            />
+          </>
+        }
+      >
+        {isWeb3InboxLoading || isCheckingXmtpAccess ? (
+          <SubscriptionPromptSkeleton />
+        ) : !web3InboxSubscription && !hasXmtpAccess ? (
+          <SubscriptionPrompt onClick={onOpen} />
+        ) : (
+          <Messages />
+        )}
+      </NotificationsSection>
+      <Modal isOpen={isOpen} onClose={onClose}>
```

Comment:
Let's try this for button labels:
<img width="476" alt="image" src="https://github.com/guildxyz/guild.xyz/assets/92519134/2cabfd78-ec36-42c5-aea7-691fed98aa5e">